release.py

```diff
@@ -315,8 +315,8 @@ def export(tag, silent=False):
             # build docs *before* we do "blurb export"
             # because docs now depend on Misc/NEWS.d
             # and we remove Misc/NEWS.d as part of cleanup for export
-            if tag.is_final or tag.level == 'rc':
-                docdist = build_docs()
+            # if tag.is_final or tag.level == 'rc':
+            #     docdist = build_docs()
```

Comment:
Remember to uncomment before merge:
```suggestion
            if tag.is_final or tag.level == 'rc':
                docdist = build_docs()
```